ci/requirements.txt

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
could we move this into `ci/mac` so its obvious why its there - also please commit the `requirements.in`

Reply:
Sure! Done!